### PR TITLE
FIX Edit style of model plot to fit in the page

### PR DESCRIPTION
**Pull request by rushic24**
# How it will look:
Readme example here :- https://huggingface.co/rushic24/TestPlaygroundSkops

Fixes https://github.com/skops-dev/skops/issues/164

### CODE REVIEW

File: skops/card/_model_card.py [hunk 1]
```diff
@@ -410,9 +410,12 @@ def _generate_card(self) -> ModelCard:
                         'clf.predict(pd.DataFrame.from_dict(config["sklearn"]["example_input"]))'
                     )
         if self.model_diagram is True:
-            model_plot: str | None = re.sub(
+            model_plot_div = re.sub(
                 r"\n\s+", "", str(estimator_html_repr(self.model))
             )
+            assert model_plot_div.count("sk-top-container") == 1
```
Comment (adrinjalali) [inline skops/card/_model_card.py:416]: production code should never have `assert` in it. Instead, you can check the value, and handle the case where it doesn't satisfy the condition.

**Review by adrinjalali — COMMENTED** [review]
Thanks for the PR @rushic24. We don't have a testing framework for these issues, but could you please create a model repo with this change so that we can see the change in action?

**Comment by adrinjalali** [comment]
You also need to apply linting (`black` and `isort`, although only `black` is relevant in this case).

You can enable them in your pre-commit hooks as explain here: https://github.com/skops-dev/skops/blob/main/CONTRIBUTING.rst#using-condamamba

**Comment by merveenoyan** [comment]
(x-posting from the issue) @rushic24 given this is about model card README in 🤗 Hub it would be nice to have a model with a lot of preprocessors in the pipeline (given it's making the plot bigger) inside a model repository that we can look at.

File: skops/card/_model_card.py [hunk 2]
```diff
@@ -410,9 +410,12 @@ def _generate_card(self) -> ModelCard:
...
+            model_plot_div = model_plot_div.replace("sk-top-container", 'sk-top-container" style="overflow: auto;')
```
Comment (merveenoyan) [inline skops/card/_model_card.py:417]: Neat 🙂 
Maybe a test that asserts this overflow style is in model card would be nice! (you can look from some model card tests for inspiration) If you need any help in writing tests please let us know hug🤗

**Review by merveenoyan — COMMENTED** [review]
Thanks a lot for working on it 🤗

**Comment by merveenoyan** [comment]
@rushic24 Lastly, can you post two repositories, one that has a big pipeline plot and another one that has a small one? (given we can't check how Hub looks on our side 😅 just to make sure both cases are fine!)

**Comment by rushic24** [comment]
Added in the Readme below https://huggingface.co/rushic24/TestPlaygroundSkops
Are the macos tests failing serious issue?

**Comment by adrinjalali** [comment]
It looks a bit narrow to me, as in, the container seems to have more space on the right side that we could use, but I'm not sure how easy it is to fix that and if it's not, I'm happy with the solution here.

**Comment by merveenoyan** [comment]
@adrinjalali it ends where the right most boundary of other components end, no? Added a photo below: image
I think it's good.

**Review by adrinjalali — APPROVED** [review]

**Pull request merged**